ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSfilemanip.cpp

```diff
@@ -107,92 +114,454 @@ void ini_section_delete(std::string section)
 
 /* OS Specific; should be moved */
 
-int file_exists(std::string fname) {
-    DWORD attributes = GetFileAttributes(fname.c_str());
-    if(attributes == 0xFFFFFFFF) {
-        return 0;
-    } else {
-        return 1;
-    }
+int file_exists(std::string fname)
+{
+    std::string str(fname);
+    std::replace(str.begin(),str.end(),'/','\\');
+
+    struct stat sb;
+    return (stat((char *)str.c_str(),&sb) == 0 &&
```

Comment:
`stat` is a POSIX function; this method should stay the way it was (or just be rewritten as `return GetFileAttributes(fname.c_str()) != INVALID_FILE_ATTRIBUTES;`). Alternatively, Win32 defines `PathFileExists(file)` which does exactly what we want (though as we've been discussing, it may be better to use the unicode equivalent of that).